apps/2I/hotel/frontend/src/app/admin/_Components/RoomDetails.tsx

```
@@ -0,0 +1,28 @@
+'use client';
+import { ArrowLeft } from 'lucide-react';
+import { Button } from '@/components/ui/button';
+import { RoomGeneralInfo } from './RoomGeneralInfo';
+import { UpcomingBookings } from './UpcomingBookings';
+import { RoomServices } from './RoomServices';
+
+type Props = {
+  roomId: string;
+};
+export const RoomDetail = ({ roomId }: Props) => {
```

ESLint

[error] 'roomId' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)